utils/config/parser.cpp

```diff
@@ -103,19 +103,6 @@ struct utils::config::parser::impl : utils::noncopyable {
 namespace {
 
 
-/// Implementation of the Lua syntax() function.
```

Comment:
Why is this gone?

Reply:
It's a duplicate definition/documentation with the one in `kyuafile.cpp`, which means that it triggers a false-ish positive with Doxygen :(.